community/modules/compute/schedmd-slurm-gcp-v5-node-group/README.md

```diff
@@ -7,6 +7,28 @@
 This module creates a node group data structure intended to be input to the
 [schedmd-slurm-gcp-v5-partition](../schedmd-slurm-gcp-v5-partition/) module.
 
+Node groups are a great way to add heterogeneity of node types to a partition,
+for example, when deploying a job with a special "manager" node with a different
+shape from the "worker" nodes. An example of multiple node groups being used can
+be found in the [slurm-gcp-v5-high-io.yaml] example blueprint.
+
+To specify nodes from a specific node group in a partition, the [`--nodelist`]
```

Comment:
After our discussion and trials this morning, I went with --mincpus where it made sense, and --nodelist otherwise. 
The error is not friendly if the node list has a larger number than the requested allocation.